src/30_Advanced/Image_Galleries_with_amp-carousel.html

```diff
@@ -125,6 +135,50 @@
       </div>
     </div>
   </amp-carousel>
+  <!-- #### Linking Carousels with amp-bind -->
+  <!--
+
+  With [amp-bind](https://github.com/ampproject/amphtml/blob/master/extensions/amp-bind/amp-bind.md) it's possible to link carousel state across multiple amp-carousels.
+
+  Whenever the carousel slide changes, it updates `selectedSlide`. This causes the `text`
+  attribute in the `<p>` tag above the carousel to update as well.
+  -->
+  <div>
+    <p> Selected slide: <span [text]="(selectedSlide + 1) + '/3'">1/3</span></p>
```

Comment:
Exactly (to remove as much duplication between binding and initial state as possible):

```
<p> Selected slide: <span [text]="(selectedSlide + 1)">1</span>/3</p>
```